app/src/main/java/ai/elimu/appstore/synchronization/AppListActivity.java

```diff
@@ -58,6 +58,7 @@ private void initInstallCompleteReceiver() {
         IntentFilter intentFilter = new IntentFilter();
         intentFilter.addAction(Intent.ACTION_PACKAGE_ADDED);
         intentFilter.addAction(Intent.ACTION_INSTALL_PACKAGE);
+        intentFilter.addAction(Intent.ACTION_PACKAGE_REMOVED);
```

Comment:
Should we create a third button for uninstalling applications?

Reply:
Should I work on it as a separate feature? I believe it makes the commits clearer.